app/scripts/modules/titus/src/serverGroup/details/resize/TitusResizeServerGroupModal.tsx

```diff
@@ -0,0 +1,305 @@
+import * as React from 'react';
+import { Modal } from 'react-bootstrap';
+import { Form, Formik, FormikContext } from 'formik';
+import { ITitusServerGroup } from 'titus/domain';
+import {
+  Application,
+  FormikFormField,
+  ICapacity,
+  IModalComponentProps,
+  MinMaxDesiredChanges,
+  ModalClose,
+  NgReact,
+  NumberInput,
+  PlatformHealthOverride,
+  ReactInjector,
+  UserVerification,
+  ValidationMessage,
+} from '@spinnaker/core';
+import { useTaskMonitor } from 'titus/serverGroup/details/resize/useTaskMonitor';
+
+const { useState, useEffect, useMemo } = React;
+
+export interface ITitusResizeServerGroupModalProps extends IModalComponentProps {
+  application: Application;
+  serverGroup: ITitusServerGroup;
+}
+
+interface ITitusResizeServerGroupCommand {
+  capacity: ICapacity;
+  serverGroupName: string;
+  instances: number;
+  interestingHealthProviderNames: string[];
+  region: string;
+}
+
+function surfacedErrorMessage(formik: FormikContext<ITitusResizeServerGroupCommand>) {
+  const capacityErrors = formik.errors.capacity || ({} as any);
+  const { min: minError, max: maxError, desired: desiredError } = capacityErrors;
+  return [minError, maxError, desiredError].find(x => !!x);
+}
+
+function SimpleMode({ formik, serverGroup, toggleMode }: IAdvancedModeProps) {
+  useEffect(() => {
+    formik.setFieldValue('capacity.min', formik.values.capacity.desired);
+    formik.setFieldValue('capacity.max', formik.values.capacity.desired);
+  }, [formik.values.capacity.desired]);
+
+  const errorMessage = surfacedErrorMessage(formik);
+
+  return (
+    <div>
+      <p>Sets min, max, and desired instance counts to the same value.</p>
+
+      <p>
+        To allow autoscaling, use the{' '}
+        <a className="clickable" onClick={toggleMode}>
+          Advanced Mode
+        </a>
+        .
+      </p>
+
+      <div className="form-group row">
+        <div className="col-md-3 sm-label-right">Current size</div>
+        <div className="col-md-4">
+          <div className="horizontal middle">
+            <input
+              type="number"
+              className="NumberInput form-control"
+              value={serverGroup.capacity.desired}
+              disabled={true}
+            />
+            <div className="sp-padding-xs-xaxis">instances</div>
+          </div>
+        </div>
+      </div>
+
+      <div className="form-group">
+        <div className="col-md-3 sm-label-right">Resize to</div>
+        <div className="col-md-4">
+          <div className="horizontal middle">
+            <FormikFormField
+              name="capacity.desired"
+              input={props => <NumberInput {...props} min={0} />}
+              layout={({ input }) => <>{input}</>}
+              touched={true}
+              onChange={() => {}}
+            />
+            <div className="sp-padding-xs-xaxis">instances</div>
+          </div>
+        </div>
+      </div>
+
+      {!!errorMessage && (
```

Comment:
Uses formik, but only shows one error message and always shows that error message in the same spot.